ci-operator/config/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main.yaml

```diff
@@ -35,18 +35,18 @@ tests:
   steps:
     test:
     - ref: janus-idp-backstage-showcase-nightly
-- as: e2e-tests-nightly-ocp-v4-15
+- as: e2e-tests-nightly-ocp-v4-16
```

Comment:
@josephca I guess we can keep the ocp 15 as it is and update the 14 to 16

Reply:
> @josephca I guess we can keep the ocp 15 as it is and update the 14 to 16

@subhashkhileri correct. The change would change the execution order of OCP 16 first then 15 which doesn't really matter, correct? :)

Reply:
right